### DISCUSSION TITLE
Beatmap-specific visual + input settings

### DISCUSSION THREAD

**Discussion by neelkarma** [ppy/osu#13409] [Ideas] [answered]
In osu!stable, the "visual settings" menu that appears before you start a beatmap allowed you to change settings for that specific beatmap, not all beatmaps. E.g. changing the background dim for one map from the visual settings menu won't affect other maps, it will only affect that map.

In osu!lazer, using the visual settings (and input settings) menu that appears before playing a beatmap allows you to change background dim and other options globally, rather than only for that beatmap. It would be nice if this acted like in osu!stable, where the settings only change for that beatmap.

<details>
<summary>(This is what I mean by "the visual settings menu that appears before you start a beatmap" btw:)</summary>
<img src="https://user-images.githubusercontent.com/59462098/121335731-33e9c800-c95e-11eb-880b-0a277a3b1ddd.png" />
</details>

<details>
<summary>(And in osu!stable:)</summary>
<img src="https://user-images.githubusercontent.com/59462098/121336832-3bf63780-c95f-11eb-8676-c4fe7e6e7bce.png" />
</details>

**Comment by gagahpangeran** [comment, marked as answer]
There has been such proposal like this in https://github.com/ppy/osu/issues/5624 and https://github.com/ppy/osu/issues/12366

Last answer is this was intentionally omitted as it has regularly caused confusion for most users on stable.
